shared/desktop/yarn-helper/font.js

```diff
@@ -133,9 +132,11 @@ function updateIconFont(web) {
     },
     (error, result) => (error ? fontsGeneratedError(error) : fontsGeneratedSuccess(web, result))
   )
+  console.log('Created new webfont')
```

Comment:
just `font` I think, or check if `web` and change the message

Reply:
sure, this was me moving it down